pypesto/objective/amici_objective.py

```diff
@@ -152,11 +155,15 @@ def __init__(self,
         # preallocate guesses, construct a dict for every edata for which we
         # need to do preequilibration
         if self.guess_steadystate:
+            if self.amici_model.ncl() > 0:
+                raise ValueError('Steadystate prediciton is not supported for'
+                                 'models with conservation laws!')
+
             if self.amici_model.getSteadyStateSensitivityMode() == \
                     amici.SteadyStateSensitivityMode_simulationFSA:
                 raise ValueError('Steadystate guesses cannot be enabled when'
                                  ' `simulationFSA` as '
-                                 'SteadyStateSensitivityMode')
+                                 'SteadyStateSensitivityMode!')
```

Comment:
I'm sorry, I don't quite get the exception, which should be raised here... When should this occur? Maybe a little comment might help...

Reply:
simulationFSA relies on correct initial state sensitivities, which we do not provide here.